website/source/docs/guides/creating-certificates.html.md

```diff
@@ -192,7 +267,149 @@ $ export CONSUL_CLIENT_KEY=cli-key.pem
 After these environment variables are correctly configured, the CLI will
 respond as expected.
 
-[cfssl]: https://cfssl.org/
+### Note on SANs for Server and Client Certificates
+
+Using `localhost` and `127.0.0.1` as `Subject Alternative Names` in server
+and client certificates allows tools like `curl` to be able to communicate with
+Consul's HTTPS API when run on the same host. Other SANs may be added during
+server/client certificates creation with `-additional-dnsname` to allow remote
+HTTPS requests from other hosts.
+
+## Configuring the Consul UI for HTTPS
+
+If your servers and clients are configured now like above, you won't be able to
+access the builtin UI anymore. We recommend that you pick a Consul agent you
+want to run the UI on and follow the instructions to get the UI up and running
+again.
+
+### Step 1: Which interface to bind to?
+
+Depending on your setup you might need to change to which interface you are
+binding because thats `127.0.0.1` by default for the UI. Either via the
+[`addresses.https`](https://www.consul.io/docs/agent/options.html#https) or
+[client_addr](https://www.consul.io/docs/agent/options.html#client_addr) option
+which also impacts the DNS server. The Consul UI is unproteced which means you
+need to put some auth in front of it if you want to make it publicly available!
```

Comment:
Should we add a note here about `enable_script_checks` and `remote exec`. They MUST NOT be enabled on these UI agents that expose HTTP to the network?

Reply:
Only HTTPS is exposed at this point. Do you think we should still mention it?

Reply:
~> Since your Consul agent is now available to the network,
`enable_script_checks` and `remote exec` must be disabled!